iib/web/api_v1.py

```diff
@@ -626,3 +632,45 @@ def add_rm_batch():
         ', '.join(processed_request_ids),
     )
     return flask.jsonify(request_jsons), 201
+
+
+@api_v1.route('/builds/merge-index-image', methods=['POST'])
+@login_required
+def merge_index_image():
+    """
+    Submit a request to merge two index images.
+
+    :rtype: flask.Response
+    :raise ValidationError: if required parameters are not supplied
+    """
+    payload = flask.request.get_json()
+    if not isinstance(payload, dict):
+        raise ValidationError('The input data must be a JSON object')
+    request = RequestMergeIndexImage.from_json(payload)
+    db.session.add(request)
+    db.session.commit()
+    messaging.send_message_for_state_change(request, new_batch_msg=True)
+
+    overwrite_target_index = _should_force_overwrite() or payload.get('overwrite_target_index')
```

Comment:
Should we use `_should_force_overwrite` on this API endpoint? I think that was introduced because the main client didn't know to provide `overwrite_from_index`. Now that this is a new API endpoint, it seems like a time to reevaluate if this is needed in this specific API endpoint.

What do you think?

Reply:
good point! I think we can remove it.

Reply:
Done